rllib/agents/dqn/dqn.py

```diff
@@ -153,33 +156,73 @@ def check_config_and_setup_param_noise(config):
                               config.get("n_step", 1))
     config["sample_batch_size"] = adjusted_batch_size
 
+    # Backward compatibility of epsilon-exploration config AND beta-annealing
+    # fraction settings (both based on schedule_max_timesteps, which is
+    # deprecated).
+    schedule_max_timesteps = None
+    if "schedule_max_timesteps" in config and \
+            config["schedule_max_timesteps"] > 0:
+        deprecation_warning(
+            "schedule_max_timesteps", "exploration.epsilon_timesteps AND "
+            "prioritized_replay_beta_annealing_timesteps")
+        schedule_max_timesteps = config["schedule_max_timesteps"]
+    if "exploration_final_eps" in config and \
+            config["exploration_final_eps"] > 0:
+        deprecation_warning("exploration_final_eps",
+                            "exploration.final_epsilon")
+        config["exploration"] = config.get("exploration",
```

Comment:
Don't need to set this since it's there by default.

Reply:
What if the user set `exploration: False`, but then still defines `exploration_final_eps`?

Reply:
I think it's fine to raise an error in that case, since they're mixing new and deprecated APIs.

Reply:
Removed.